engine/src/main/java/com/redhat/cloud/notifications/models/EmailAggregation.java

```diff
@@ -26,9 +26,6 @@ public class EmailAggregation extends CreationTimestamped {
     @JsonProperty(access = READ_ONLY)
     private Integer id;
 
-    @Size(max = 50)
-    private String accountId;
```

Comment:
This field will be removed from Postgres after this PR has been deployed on prod.